conftest.py

```diff
@@ -68,25 +88,25 @@ def instance_path(renku_path, monkeypatch):
         yield renku_path
 
 
-@pytest.fixture()
+@pytest.fixture(scope="function")
```

Comment:
I believe default scope is `function` and we don't need to specify it here and other places in this file.

Reply:
I added them everywhere because I originally didn't trust it and thought it might inherit from other fixtures, but didn't remove it after that turned out to not be the case.